pulsar-client-cpp/lib/lz4/lz4.h

```diff
@@ -402,4 +402,4 @@ int LZ4_decompress_safe_withPrefix64k(const char *src, char *dst, int compressed
 LZ4_DEPRECATED("use LZ4_decompress_fast_usingDict() instead")
 
 int LZ4_decompress_fast_withPrefix64k(const char *src, char *dst, int originalSize);
-}
\ No newline at end of file
+}  // namespace pulsar
```

Comment:
Please don't comment like this

Reply:
+1, this change is unnecessary. Please remove it.